jest

```diff
@@ -0,0 +1,12 @@
+#!/usr/bin/env node
+/**
+ * Copyright (c) 2014, Facebook, Inc. All rights reserved.
+ *
+ * This source code is licensed under the BSD-style license found in the
+ * LICENSE file in the root directory of this source tree. An additional grant
+ * of patent rights can be found in the PATENTS file in the same directory.
+ */
+
+'use strict';
+
+require('./packages/jest-cli/bin/jest');
```

Comment:
I added this "binary" because I have to type this out so often from within the Jest folder and from other folders. This allows me to type `node-4 jest` and it will work nicely.